tests/sentry/sentry_metrics/test_id_generator.py

```diff
@@ -8,8 +8,8 @@ def test_get_id() -> None:
     # Function returns a different ID each time it's called
     assert get_id() != get_id()
 
-    # IDs fit in 64 bits
-    assert get_id() < pow(2, 64)
+    # IDs fit in 63 bits (leading bit must be a zero)
+    assert get_id() < pow(2, 63)
 
     # Starts with 0100 (version)
```

Comment:
maybe clarify that this is the zero bit + the version. Also the comment for `test_get_id_time_since needs` to be updated

Reply:
updated